docs/en/serving/restful_api.md

```diff
@@ -148,6 +148,16 @@ You can also test restful-api through webui.
 lmdeploy serve gradio api_server_url --server-name ${gradio_ui_ip} --server-port ${gradio_ui_port}
 ```
 
+### webui through OpenAOE
+
+You can use [OpenAOE](https://github.com/InternLM/OpenAOE) for seamless integration with LMDeploy.
+
+```shell
+pip install -U openaoe 
+openaoe -f /path/to/your/config-template.yaml
```

Comment:
May provide a `config-template.yaml` or a link to get it.

Reply:
I see. [There](https://github.com/InternLM/OpenAOE/blob/main/docs/tech-report/config-template.md) it is.